.circleci/config.yml

```diff
@@ -11,6 +11,17 @@ jobs:
     working_directory: /go/src/github.com/codilime/floodgate
     steps:
       - checkout
+      - run:
+          name: Setup swagger-codegen
+          command: |
+            sudo apt update
+            sudo apt install --no-install-recommends openjdk-11-jre-headless
+            SWAGGER_VERSION=$(cat gateapi/.swagger-codegen/VERSION)
+            wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${SWAGGER_VERSION}/swagger-codegen-cli-${SWAGGER_VERSION}.jar -O swagger-codegen-cli.jar
```

Comment:
What do you think about calculating sha1 sum after downloading?

Reply:
Good idea, added it.